src/vsl/parser/grammar/assignment.ne

```diff
@@ -2,16 +2,21 @@
 @include "primitives.ne"
 
 # Parses assignment statements
-AssignmentStatement -> AssignmentType _ TypedIdentifier ( _ "=" _ Expression {% nth(3) %}):? {%
-    (d, l) => new t.AssignmentStatement(
-        d[0] === "let" ? t.AssignmentType.Constant : t.AssignmentType.Variable,
-        d[2],
-        d[3],
-        l
-    )
-%}
+AssignmentStatement
+   -> AssignmentType _ TypedIdentifier ( _ "=" _ Expression {% nth(3) %}):? {%
+        (data, location) => {
+            var type;
+            if (data[0] === "let") {
```

Comment:
I could probably use a ternary, but my main argument here is that I'm more concerned about readability than saving a couple lines. We want others to be able to read it, not just ourselves, since it can be very easy to put yourself in a position where you can read your code perfectly but most other developers can't. In that sense, this is a precautionary action I usually take to keep my code less likely to jumble itself up into an unreadable mass of multicolored text.